examples/cosmos/chain_core_test.go

```diff
@@ -657,7 +657,7 @@ func testStaking(ctx context.Context, t *testing.T, chain *cosmos.CosmosChain, u
 		del, err := chain.StakingQueryDelegationsTo(ctx, val)
 		require.NoError(t, err)
 		require.NotEmpty(t, del)
-		require.EqualValues(t, "1000", del[1].Balance.Amount.String())
+		require.EqualValues(t, "1000", del[0].Balance.Amount.String())
```

Comment:
This is not being very deterministic for testing. May be due to unique addresses ever time

Reply:
resolved in 99c1ff2f33de2d5e076c8b54de949aeb70c089c1 (get specific delegation, iter all and find 1 that matches the amount 1000. then continue)